sympy/integrals/transforms.py

```diff
@@ -1162,6 +1162,90 @@ def sbs(expr):
         aux = _simplifyconds(aux, s, a)
     return _simplify(F.subs(s, s_), simplify), sbs(a), _canonical(sbs(aux))
 
+def laplace_transform_ode(eqs, t, s, funcs):
+    r"""
+    Function that can solve some ordinary differential equation systems by
+    Laplace transform.
+    """
+    from sympy import solve
+    eqs_laplace = []
+    for eq in eqs:
+        xl = laplace_transform(eq, t, s)
+        for _f, _F, _ic in funcs:
+            xl = laplace_transform_replace_ic(xl, _f, _F, t, s, _ic)
+        eqs_laplace.append(xl)
+    Funcs = []
+    for _f, _F, _ic in funcs:
+        Funcs.append(_F(s))
+    sols_laplace = solve(eqs_laplace, Funcs)
```

Comment:
I think it would be better to return the transform of the expressions unsolved. Then the user can call solve if they want to.

It would also be good to handle equations by transforming each side.

Reply:
Done. It now also has a flag solve_it that defaults to `False`.

If you're happy, I'll write a doc and test cases.